yolox/utils/mlflow_logger.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env python3
+# Copyright (c) Megvii Inc. All rights reserved.
+# Please read docs/mlflow_integration.md for more details.
+"""
+Logging training runs with hyperparameter, datasets and trained models to MlFlow.
+Mlflow support Model Tracking, Experiment Tracking, and Model Registry.
+It can be hosted on-premises or in all the major cloud provider or with databricks also.
+Please read docs/mlflow_integration.md for more details.
+
+For changing default logging Behaviour you can change mlflow environment variables:
+    https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html
+
+For more information, please refer to:
+https://mlflow.org/docs/latest/introduction/index.html
+"""
+import importlib.metadata
+import importlib.util
+import json
+import os
+from collections.abc import MutableMapping
+import packaging.version
+
+from loguru import logger
+import torch
+from yolox.utils import is_main_process
+class MlflowLogger:
```

Comment:
Did you lint with this repo's setting correctly?  It should be two blank line before the class  definition.

You might check your lint result here: https://github.com/Megvii-BaseDetection/YOLOX/actions/runs/9796205233/job/27068037255?pr=1773

Reply:
Yes, linted again to remove these lint issue.